### PR TITLE
Add isFromCustomerChatPlugin in Messenger event

### LINKED ISSUE #56: Add support to detecting message origin
Should we name it to `isFromCustomerChatPlugin`?

https://developers.facebook.com/docs/messenger-platform/discovery/customer-chat-plugin#detect

### PR DESCRIPTION
**Pull request by xxhomey19**
close #56

### CODE REVIEW

File: src/context/MessengerEvent.js [hunk 1]
```diff
@@ -41,12 +41,17 @@ type MediaAttachment = {
 
 type Attachment = MediaAttachment | FallbackAttachment;
 
+type Tags = {
```
Comment (chentsulin) [inline src/context/MessengerEvent.js]: Tag

File: src/context/MessengerEvent.js [hunk 2]
```diff
@@ -41,12 +41,17 @@ type MediaAttachment = {
...
+  [key: string]: string,
```
Comment (chentsulin) [inline src/context/MessengerEvent.js]: source

File: src/context/MessengerEvent.js [hunk 3]
```diff
@@ -41,12 +41,17 @@ type MediaAttachment = {
...
+};
+
 export type Message = {
   is_echo?: boolean,
   text?: string,
   sticker_id?: number,
   quick_reply?: QuickReply,
   attachments?: Array<Attachment>,
+  tags?: Array<Tags>,
```
Comment (chentsulin) [inline src/context/MessengerEvent.js]: Array<Tag>

File: src/context/MessengerEvent.js [hunk 4]
```diff
@@ -612,4 +617,19 @@ export default class MessengerEvent implements Event {
   get takeThreadControl(): ?TakeThreadControl {
     return this._rawEvent.take_thread_control || null;
   }
+
+  /**
+   * Determine if the event is from custome chat plaugin.
```
Comment (tw0517tw) [inline src/context/MessengerEvent.js]: plaugin => plugin

**Pull request merged**